package/boot/uboot-rockchip/Makefile

```
@@ -278,6 +278,13 @@ define U-Boot/nanopi-r5s-rk3568
     friendlyarm_nanopi-r5s
 endef
 
+define U-Boot/xgp-rk3568
```

```suggestion
define U-Boot/xiguapi-v3-rk3568
```

the config name should match the dt compatible (`[<vendor>,]<board>`)
rename and sort alphabetically